ctapipe/image/tests/test_hillas.py

```diff
@@ -137,7 +139,9 @@ def test_with_toy():
             assert u.isclose(result.y, y, rtol=0.1)
 
             assert u.isclose(result.width, width, rtol=0.1)
+            assert u.isclose(result.width_uncertainty, width_uncertainty, rtol=1)
```

Comment:
1 is a rather giant rtol, I see that we don't really have a nice way to come up with the `correct` value other than just testing the implementation with the implementation. So It might be ok to leave it as is.

Reply:
I have made some distributions of the uncertainties using the toy model and I have fine-tuned the expected values and tolerance for the unit test.